parm/archive/gfsa.yaml.j2

```diff
@@ -46,6 +46,17 @@ gfsa:
         - "{{ COMIN_OBS | relpath(ROTDIR) }}/{{ head }}aerorawobs"
         {% endif %}
 
+        # Snow analysis data
+            {% if DO_JEDISNOWDA %}
+                {% for itile in range(1,7) %}
+        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/snowinc.{{ cycle_YMD }}.{{ cycle_HH }}0000.sfc_data.tile{{ itile }}.nc"
+        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ cycle_HH }}0000.sfc_data.tile{{ itile }}.nc"
```

Comment:
I think we said we don't need the analysis fields archived, just the increment for the GFS cycle? Right @ClaraDraper-NOAA? also, indentation looks a bit off for the Jinja, otherwise this looks good

Reply:
Remove the analysis fields and fix the indentation as suggested.

Reply:
I just checked the gfs arch, but didn't find the `gfs_restartb.yaml.j2` available to usually be used holding the sfc background from the previous cycle. Therefore, we need to keep the analysis fields here. @CoryMartin-NOAA Do you agree with this?